components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/ConfigurationFacade.java

```diff
@@ -59,12 +68,48 @@ public static ConfigurationFacade getInstance() {
         return instance;
     }
 
+    /**
+     * Returns the sequence config with given parameters.
+     * @param reqType
+     * @param relyingParty
+     * @param tenantDomain
+     * @return
+     * @throws FrameworkException
+     * @deprecated Please use  #getSequenceConfig(AuthenticationContext, Map) instead.
+     */
+    @Deprecated
     public SequenceConfig getSequenceConfig(String reqType, String relyingParty, String tenantDomain)
             throws FrameworkException {
 
         // Get SP config from SP Management component
-        return UIBasedConfigurationBuilder.getInstance().getSequence(reqType,
-                                                                     relyingParty, tenantDomain);
+        return UIBasedConfigurationBuilder.getInstance().getSequence(reqType, relyingParty, tenantDomain);
+    }
+
+    /**
+     * TODO: Move this to better place
+     * @param context
+     * @param parameterMap
+     * @return
+     * @throws FrameworkException
```

Comment:
Please add parameter descriptions here. Otherwise, these pops up as yellow lines in IntelliJ IDEA!!